UPGRADE-4.2.md

```diff
@@ -155,6 +155,7 @@ Translation
 -----------
 
  * The `TranslatorInterface` has been deprecated in favor of `Symfony\Contracts\Translation\TranslatorInterface`
+ * The `Translator::transChoice()` has been deprecated in favor of using `Translator::trans()` with intl message format 
```

Comment:
I choose to use the implementation (`Translator`) here. We already say that the full interface is deprecated.